.changeset/perfect-beers-grab.md

```diff
@@ -0,0 +1,21 @@
+---
+'@didtools/key-secp256k1': minor
+'@didtools/key-webcrypto': minor
+'jest-environment-ceramic': minor
+'key-did-provider-ed25519': minor
+'key-did-resolver': minor
+'pkh-did-resolver': minor
+'@didtools/key-webauthn': minor
+'@didtools/pkh-ethereum': minor
+'did-session': minor
+'@didtools/pkh-solana': minor
+'@didtools/pkh-stacks': minor
+'@didtools/pkh-tezos': minor
+'@didtools/multidid': minor
+'@didtools/codecs': minor
+'@didtools/cacao': minor
+'dids': minor
```

Comment:
Should this be major for the packages already past v1? Not sure there's anything breaking, but I'm not sure there isn't either...

Reply:
given no api changes, I think this is just a binary incompatible or minor change.